android/src/main/kotlin/com/simform/audio_waveforms/AudioWaveformsPlugin.kt

```diff
@@ -113,6 +113,11 @@ class AudioWaveformsPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
                     result.error(Constants.LOG_TAG, "Player key can't be null", "")
                 }
             }
+            Constants.releaseAudioPlayerResource -> {
+                val key = call.argument(Constants.playerKey) as String?
+                audioPlayers[key]?.release(result)
+                recorder = null
```

Comment:
Why are we dereferencing recorder here?

Reply:
By mistakenly added. Actually was refer the stop method of recording so that's why its added here.